### PR TITLE
Make header consistent with Landing page

### PR DESCRIPTION
**Pull request by Andkoo**

### CODE REVIEW

File: assets/main.js [hunk 1]
```diff
@@ -1,48 +1,48 @@
 // document query .signup-button
-
 const signup_button = document.querySelector('.signup-button')
 const signup_input = document.querySelector('.signup-input')
-console.log('signup_button', signup_button)
```
Comment (Andkoo) [inline assets/main.js:5]: This whole section which has been "removed" was just wrapped in a `if (signup_button)` condition to fix console error about missing element.

File: assets/main.js [hunk 2]
```diff
@@ -57,3 +57,63 @@ function validateEmail(email) {
   var re = /\S+@\S+\.\S+/
   return re.test(email)
 }
+
+// Header scroll interaction
```
Comment (Andkoo) [inline assets/main.js:61]: This is where the actually relevant code starts

File: _includes/footer.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <footer>
   <a class="logo" href="/" rel="author">
-    <img alt="Logo" height="57" src="/assets/images/logo.png" width="253"/>
+    <img src="/assets/images/logo.svg" width="256" height="57" alt="Mangata" />
```
Comment (devdanco) [inline _includes/footer.html:3]: you can use **require("/assets/images/logo.svg").default** in src .. I read somewhere that it is better way to load images.

**Review by devdanco — CHANGES_REQUESTED** [review]

File: _includes/footer.html (hunk 3 above)
Comment (Andkoo) [reply _includes/footer.html:3]: The blog doesn't use webpack and/or react. I see your point that putting images as base64 will display them faster than loading them via requests, however that's not really possible here.

Comment (devdanco) [reply _includes/footer.html:3]: ok  great :)

**Review by devdanco — APPROVED** [review]

**Pull request merged**